static/css/styles.css

```
@@ -0,0 +1,273 @@
+      padding: 0;
+      box-sizing: border-box;
+    }
+
+
+    body {
+      background-image: url("https://cdn.jsdelivr.net/gh/Terry-BrooksJr/the-dozens-django@cfed15844ea94edfc7e4f61a1fb60f92570aeb68/static/assets/background.svg");
+      background-repeat: no-repeat;
+      background-color: #d1d937;
+      background-size: cover;
+
+      font-family: "YoMamaBody";
```

Biome (2.1.2)

[error] Generic font family missing.

Consider adding a generic font family as a fallback.
For examples and more information, see the MDN Web Docs


(lint/a11y/useGenericFontNames)